src/calculators/abv.rs

```diff
@@ -1,37 +1,44 @@
 extern crate clap;
 use clap::{App, Arg, ArgMatches, SubCommand};
+use crate::AppSubCommand;
 
-fn calculate_abv(og: f32, fg: f32) -> f32 {
-    (og - fg) * 131.25
-}
+pub struct Abv;
+
+impl AppSubCommand for Abv {
+    fn add_subcommand<'a, 'b>(&self, app: App<'a, 'b>) -> App<'a, 'b>{
+        let ret = app.subcommand(SubCommand::with_name("abv")
+            .version("0.1")
+            .author("Heikki Hellgren (heiccih@gmail.com)")
+            .about("Calculates ABV from original and final gravity")
+            .arg(Arg::with_name("og")
+                 .short("o")
+                 .long("og")
+                 .value_name("OG")
+                 .help("Original gravity")
+                 .required(true)
+                 .takes_value(true))
+            .arg(Arg::with_name("fg")
+                 .short("f")
+                 .long("fg")
+                 .value_name("FG")
+                 .help("Final gravity")
+                 .required(true)
+                 .takes_value(true)));
 
-pub fn add_subcommand<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b>{
-    let ret = app.subcommand(SubCommand::with_name("abv")
-        .version("0.1")
-        .author("Heikki Hellgren (heiccih@gmail.com)")
-        .about("Calculates ABV from original and final gravity")
-        .arg(Arg::with_name("og")
-             .short("o")
-             .long("og")
-             .value_name("OG")
-             .help("Original gravity")
-             .required(true)
-             .takes_value(true))
-        .arg(Arg::with_name("fg")
-             .short("f")
-             .long("fg")
-             .value_name("FG")
-             .help("Final gravity")
-             .required(true)
-             .takes_value(true)));
+        ret
+    }
```

Comment:
for aesthetic reasons, you can leave out `let ret = `, the semicolon, and the `ret` :) It worked for the priming calculator.